### PR TITLE
Processing data from a form

### PR DESCRIPTION
**Pull request by BognaLew**

### CODE REVIEW

**Review by AaronWantoch — APPROVED** [review]

File: .github/linters/.python-lint
Comment (Lenetis) [inline .github/linters/.python-lint:1]: Maybe move this to another PR? It doesn't have anything to do with forms

File: src/main.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from flask import Flask, render_template, redirect, url_for
+from forms import RegisterForm, OrderSubscriptionForm
```
Comment (Lenetis) [inline src/main.py]: The same import is repeated on line 7

**Review by Lenetis — APPROVED** [review]
